util/grappa.mk

```diff
@@ -0,0 +1,70 @@
+#############################################################################
+# Grappa prototype make file
+
+# This sets variables such that GNU Make's implicit rules will work
+# for compiling basic Grappa programs with compilers supporting
+# GNU-compatible flags. The file is automatically generated by CMake
+# when Grappa is configured and built.
+
+# To use this file, write a Makefile including lines like these:
+# > include ${CMAKE_BINARY_DIR}/util/grappa.mk
+# > grappa_app: grappa_app.o
+
+# This uses Make's implicit rules, so "make grappa_app" will look for
+# grappa_app.cpp, compile it into grappa_app.o, and link grappa_app.o
+# into a binary named grappa_app.
+
+# The GNU Make implicit compilation recipe for C++ files which turns a
+# file n.cpp/n.cc/n.C into n.o looks like: 
+# $(CXX) $(CPPFLAGS) $(CXXFLAGS) -c -o n.o n.cpp
+
+# The GNU Make implicit link recipe that turns n.o into an executable n is:
+# $(CC) $(LDFLAGS) n.o $(LOADLIBES) $(LDLIBS) -o n
```

Comment:
Curiously, no: https://www.gnu.org/software/make/manual/html_node/Catalogue-of-Rules.html#Catalogue-of-Rules